source/features/delete-review-comments-faster.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import {observe} from 'selector-observer';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function deleteComment(comment: HTMLElement): void {
+	select('.dropdown-menu .js-comment-delete > button', comment)!.click();
+}
+
+function onButtonClick(event: delegate.Event): void {
+	const comment = (event.target as HTMLElement).closest<HTMLElement>('.js-comment')!;
+	const dropdownFragment = select('include-fragment.SelectMenu-loading', comment);
+	if (dropdownFragment) {
+		select('.timeline-comment-actions > details:last-of-type', comment)!.dispatchEvent(new Event('mouseover'));
+		dropdownFragment.addEventListener('load', () => {
+			deleteComment(comment);
+		});
+		return;
+	}
+
+	deleteComment(comment);
+}
+
+function init(): void {
+	delegate(document, '.rgh-review-comment-delete-button', 'click', onButtonClick);
+	observe('.review-comment > .unminimized-comment .btn-primary[type="submit"]', {
```

Comment:
For some reason this doesn't work:
```suggestion
	observe('.review-comment > .unminimized-comment .btn-primary[type="submit"]:not(.rhg-delete-button-added)', {
```

Reply:
Try again  😁 I don't see why it wouldn't work, the selector appears to be valid

Reply:
Got it, it was me who forgot to `onetime()` the `init` :man_facepalming: